core/src/main/java/greencity/controller/HabitController.java

```diff
@@ -5,11 +5,7 @@
 import greencity.annotations.ValidLanguage;
 import greencity.constant.HttpStatuses;
 import greencity.dto.PageableDto;
-import greencity.dto.habitstatistic.AddHabitStatisticDto;
-import greencity.dto.habitstatistic.HabitDictionaryTranslationsDto;
-import greencity.dto.habitstatistic.HabitItemsAmountStatisticDto;
-import greencity.dto.habitstatistic.HabitStatisticDto;
```

Comment:
`HabitStatisticDto` should stay. The constructor that depends on entity should be replaced by mapper.

Reply:
Most probably the same for other DTOs that are used in controllers. Only DTOs that are mapped as query result can go to dao level.